services/src/main/proto/grpc/instrumentation/v1alpha/monitoring.proto

```diff
@@ -76,39 +74,67 @@ service Monitoring {
-
-  View rpc_client_errors = 1;
-  View rpc_client_completed_rpcs = 2;
-  View rpc_client_started_rpcs = 3;
-  View rpc_client_elapsed_time = 4;
-  View rpc_client_server_elapsed_time = 5;
-  View rpc_client_request_bytes = 6;
-  View rpc_client_response_bytes = 7;
-  View rpc_client_request_count = 8;
-  View rpc_client_response_count = 9;
-  View rpc_server_errors = 10;
-  View rpc_server_completed_rpcs = 11;
-  View rpc_server_server_elapsed_time = 12;
-  View rpc_server_request_bytes = 13;
-  View rpc_server_response_bytes = 14;
-  View rpc_server_request_count = 15;
-  View rpc_server_response_count = 16;
-  View rpc_server_elapsed_time = 17;
+  StatsResponse rpc_client_errors = 1;
+  StatsResponse rpc_client_completed_rpcs = 2;
+  StatsResponse rpc_client_started_rpcs = 3;
+  StatsResponse rpc_client_elapsed_time = 4;
+  StatsResponse rpc_client_server_elapsed_time = 5;
+  StatsResponse rpc_client_request_bytes = 6;
+  StatsResponse rpc_client_response_bytes = 7;
+  StatsResponse rpc_client_request_count = 8;
+  StatsResponse rpc_client_response_count = 9;
+  StatsResponse rpc_server_errors = 10;
+  StatsResponse rpc_server_completed_rpcs = 11;
+  StatsResponse rpc_server_server_elapsed_time = 12;
+  StatsResponse rpc_server_request_bytes = 13;
+  StatsResponse rpc_server_response_bytes = 14;
+  StatsResponse rpc_server_request_count = 15;
+  StatsResponse rpc_server_response_count = 16;
+  StatsResponse rpc_server_elapsed_time = 17;
   //TODO(ericgribkoff) Add minute-hour interval stats.
 }
 
+// This message is sent when requesting a set of stats (Census Views) from
+// a client system, as part of the MonitoringService API's.
 message StatsRequest {
-  // TODO(aveitch): Complete definition of this type
+  // An optional set of ViewDescriptor names. Only Views using these
+  // descriptors will be sent back in the response. If no names are provided,
+  // then all Views present in the client system will be included in every
+  // response. If measurement_names is also provided, then Views matching the
+  // intersection of the two are returned.
+  // TODO(aveitch): Consider making this a list of regexes or prefix matches in
+  // the future.
+  repeated string view_names = 1;
+
+  // An optional set of MeasurementDescriptor names. Only Views using these
+  // descriptors will be sent back in the response. If no names are provided,
+  // then all Views present in the client system will be included in every
+  // response. If view_names is also provided, then Views matching the
+  // intersection of the two are returned.
+  // TODO(aveitch): Consider making this a list of regexes or prefix matches in
+  // the future.
+  repeated string measurement_names = 2;
+
+  // By default, the MeasurementDescriptors and ViewDescriptors corresponding to
+  // the Views that are returned in a StatsResponse will be included in the
+  // first such response. Set this to true to have them not sent.
+  bool dont_include_descriptors_in_first_response = 3;
```

Comment:
This is coming from https://github.com/grpc/grpc-proto/pull/3 as the source of truth, and is just copied here.